### PR TITLE
Borg spinning no longer stuns.

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Borg spinning knocks down for 3 seconds instead of stunning for 6.

## Why It's Good For The Game

Meme mechanics shouldn't even be possible to be viable as a combat tool.

## Changelog
:cl:
balance: cyborgs spinning off riders now knocks down for 3 seconds instead of stunning for 6
/:cl:

### CODE REVIEW

**Comment by 4dplanner** [comment]
Would it not be better to just give a long enough timer that you can't combat buckle someone to yourself? Betraying someone who chooses to ride you is a good mechanic

**Comment by PKPenguin321** [comment]
>Betraying someone who chooses to ride you is a good mechanic

Has this literally ever been done even once
Everyone knows spin stunning is just a combat-borg strategy, even if the situation you describe has happened then a combat stun has happened 100x as much
This shit has no good reason to stun

**Comment by ATH1909** [comment]
> > Betraying someone who chooses to ride you is a good mechanic
> 
> Has this literally ever been done even once

Yes, by me.

**Comment by 4dplanner** [comment]
Further, instant buckle to knockdown spin would still be a viable combat mechanic anyway with this PR, wouldn't it?

**Comment by ATH1909** [comment]
> Further, instant buckle to knockdown spin would still be a viable combat mechanic anyway with this PR, wouldn't it?

imsxz's PR got merged, so borg buckling is no longer instant.

**Comment by 4dplanner** [comment]
Whoops, missed that.

**Comment by carshalash** [comment]
Is this still going to be a thing after the other one got through?

**Comment by TheSilverNuke** [comment]
I play AI frequently and I have to say, this is the dumbest PR I've seen for a bit. Borgs have two ways to stun; flashes (there was a PR to remove flashes awhile back too) and spin stuns. Every other way to stun is harmful and conflicts with the Asimov AI lawset, or is too convoluted to do in a timely manner.

Basically what people seem to be out to do, is to tie the hands of silicons even more than is already present. I would like to counter by saying that if you want to allow this PR to be merged, to provide an alternate means of stunning that silicons can do on the regular. Otherwise shove off.

**Comment by cacogen** [comment]
I don't think the spin emote was ever intended to be used in combat, and since borg buckling was introduced it's always been dumb that borgs have no other way of unbuckling people than throwing them into walls. Just let them click themselves without a module selected to unbuckle people. Spin throws (i.e. fun) don't have to go, just the OP bullshit they've become as a result of unforeseen uses (emergent gameplay).

**Comment by TheSilverNuke** [comment]
> 
> 
> I don't think the spin emote was ever intended to be used in combat, and since borg buckling was introduced it's always been dumb that borgs have no other way of unbuckling people than throwing them into walls. Just let them click themselves without a module selected to unbuckle people. Spin throws (i.e. fun) don't have to go, just the OP bullshit they've become as a result of unforeseen uses (emergent gameplay).

Then if you think spin wasn't an intended stun feature code an alternative instead of flat out deleting it. You need to provide an alternate method before you remove the ones we have.

**Comment by CRITAWAKETS** [comment]
Spinstunning always felt to me like it belonged in the borg arsenal. Something for more experienced players to learn and use. Its easier to flash anyway if they dont have sunglasses.

Also Imszx's PR kinda made this one obsolete. It doesnt really have the same power anymore since your target has to not move for a bit.

**Comment by cacogen** [comment]
@TheSilverNuke unless it's changed i don't think you can push your own shit

**Comment by PKPenguin321** [comment]
>Then if you think spin wasn't an intended stun feature code an alternative instead of flat out deleting it. You need to provide an alternate method before the ones we have are removed.

What? Why would you need to put an alternative for stunning if it wasn't supposed to stun in the first place?

**Comment by TheSilverNuke** [comment]
> What? Why would you need to put an alternative for stunning if it wasn't supposed to stun in the first place?

Well for one I doubt it wasn't intended from the start because of the fact that it specifically allows borgs to throw people vast distances and stun them, and has done so for some time without any input. For two, basically any silicon player will tell you that it's an incredibly critical feature that absolutely needs to stay implemented.

When you run across a murderboning tider with sunglasses and you can't spin stun them or flash them, tell me how you will stop them if you can't harm them at all. It's essentially like playing the game with no arms.

**Comment by cacogen** [comment]
Just give them all a stun baton instead of a flash or something

**Comment by TheSilverNuke** [comment]
I would love a stun baton to be honest. But emagged engiborgs already get those, so you would have to give them a neat new toy for being emagged as well. Additionally that just lowers the skill level needed to play borg so it makes the previous attempts to nerf silicons redundant. I won't complain if you push it through though.

**Comment by actioninja** [comment]
> You need to provide an alternate method before the ones we have are removed.

No we don't.

**Comment by ATH1909** [comment]
> I would love a stun baton to be honest. 

Shh, you're making the rest of us anti-spin-stun-nerf-ers look like extremists. The maintainers won't even let us give peacekeeper borgs pepper spray, so I doubt that they'd approve of talk of anything related to secborgs.

**Comment by TheSilverNuke** [comment]
> 
> 
> > You need to provide an alternate method before the ones we have are removed.
> 
> No we don't.

Yes you do. It's called being responsible with your code. You can't just leave a game incomplete intentionally. The only people who are able to do that are scammers and AAA game companies (whats the difference?)

**Comment by actioninja** [comment]
> Yes you do. It's called being responsible with your code. You can't just leave a game incomplete intentionally.

The implication here is that the removal of something leaves the game incomplete, which is incorrect.

**Comment by TheSilverNuke** [comment]
So by that logic if we just rip atmos out of the game it won't leave the game incomplete? Alright sounds good. Go ahead and delete atmos. Who needs air anyway?

**Comment by actioninja** [comment]
That's a strawman and you know it.

**Comment by TheSilverNuke** [comment]
Maybe, but it doesn't make what I am saying any less correct. Provide an alternative to the stun spin or don't merge it. I will continue to voice my opinion on this until it's done.

On an unrelated note, what brought about this nerf? What makes you feel that this is necessary?

**Comment by actioninja** [comment]
> Maybe, but it doesn't make what I am saying any less correct. Provide an alternative to the stun spin or don't merge it. I will continue to voice my opinion on this until it's done.

Ok.

> On an unrelated note, what brought about this nerf? What makes you feel that this is necessary?

Borgs shouldn't get a free repeatable 6 second long stun from something that wasn't even intended to be able to be weaponized.

**Comment by TheSilverNuke** [comment]
It's been working fine thus far, and with very few complaints. In fact, the stun spin is one of the few weapons in a borg's arsenal that they are able to use effectively without harm. The ONLY other one is the flash, and simply sunglasses, welding goggles, welding helmet, etc, will counter that. 

I would say that there are more cons to playing as a borg than pros. Borgos have so many counters right now it's straight ridiculous. Electric shock from a defib, flash, flashbang, EMP, getting tossed around, any and all ranged weaponry, any decent melee weapon, etc.

Now to counter all of those, again the borgs on default Asimov have either a flash or the spin stun. That is it. The rest are situational or are too convoluted to use. Janiborgs for example can mop stuff, mediborgs can buckle to a bed, peacekeepers have a harm alarm, default borgs have cuffs, engiborg has an RCD. You COULD drag a can of N2O as a borg 24/7 in the small case that there is an outbreak of human harm but again that's pretty convoluted.

Borgs are in a pretty balanced state right now, and this PR just straight runs them into the ground. I hope you realize that after reading this.

**Comment by actioninja** [comment]
> I would say that there are more cons to playing as a borg than pros.

lol

**Comment by carshalash** [comment]
Just noticed the name of your branch, cringe.

**Comment by actioninja** [comment]
ok silicon main

**Comment by TheSilverNuke** [comment]
You would think Silicon mains are in the best position to give feedback on gameplay elements regarding borgs and AI.

**Comment by actioninja** [comment]
ok silicon main

**Comment by ninjanomnom** [comment]
You would think that but unfortunately the majority of [x] mains generally have the goal of increasing the strength of their role. It's the discretion of maintainers to decide what forms of information they want to use when making decisions about changes.

**Comment by carshalash** [comment]
🤔 what does being a silicon main have to do with your branch name?

**Review by ninjanomnom — APPROVED** [review]
The purpose of this mechanic is as a way to get people off of you and for fun. I don't see this being viable in combat as intended or healthy.

**Comment by TheSilverNuke** [comment]
I would like to disagree and reference my above position regarding the Spin stun.

>It's been working fine thus far, and with very few complaints. In fact, the stun spin is one of the few weapons in a borg's arsenal that they are able to use effectively without harm. The ONLY other one is the flash, and simply sunglasses, welding goggles, welding helmet, etc, will counter that.

I would say that there are more cons to playing as a borg than pros. Borgos have so many counters right now it's straight ridiculous. Electric shock from a defib, flash, flashbang, EMP, getting tossed around, any and all ranged weaponry, any decent melee weapon, etc.

Now to counter all of those, again the borgs on default Asimov have either a flash or the spin stun. That is it. The rest are situational or are too convoluted to use. Janiborgs for example can mop stuff, mediborgs can buckle to a bed, peacekeepers have a harm alarm, default borgs have cuffs, engiborg has an RCD. You COULD drag a can of N2O as a borg 24/7 in the small case that there is an outbreak of human harm but again that's pretty convoluted.

Borgs are in a pretty balanced state right now, and this PR just straight runs them into the ground. I hope you realize that after reading this.

-Edit- Oh I forgot, there was a PR recently that slowed the buckle time of people to borgos too. That takes a full second.

-Edit 2- Also if you don't want to get stun spinned by borgos literally all you have to do is to keep both hands full. That's pretty easy, one hand has a flash, one hand has a brute force weapon. GG.

**Comment by FancyCyclops** [comment]
The code is not a tool to be used to robust people that you can't deal with.

**Comment by actioninja** [comment]
Guess again idiot, I'm a coder so I don't even play the game. Impossible to be robusted if the last time I logged in was 8 months ago.

**Comment by FancyCyclops** [comment]
Then use the players to get feedback, rather than playing with the code how YOU and your buttbuddies like it.

**Comment by actioninja** [comment]
Nah.

**Comment by TheSilverNuke** [comment]
TIL that coders literally don't give a fuck about the playerbase or what they think, and will push through a PR despite a well written argument against it, and will just reply with "lol."

The complete indifference you show is infuriating. Watch this be some grand master plot to get players to stop playing on /TG

**Comment by actioninja** [comment]
Ok, bye pedo.

**Comment by FancyCyclops** [comment]
They need players to make money. They just need a more loyal playerbase. And attention.

**Comment by FancyCyclops** [comment]
Ok coder.

**Comment by ninjanomnom** [comment]
Hey I'm not a butt kind of guy, I'm more into the handholding

**Comment by FancyCyclops** [comment]
Shut up kid.

**Comment by TheVekter** [comment]
> Ok, bye pedo.

image

**Comment by TheSilverNuke** [comment]
What the hell has happened to TG

**Comment by Rain7x** [comment]
yea it's really gone downhill merging PR's with mean branch names

**Comment by TheSilverNuke** [comment]
> 
> 
> yea it's really gone downhill merging PR's with mean branch names

If you think that's what this is about you're missing the point. He's completely ignored any comment of mine, and has only responded with taunts and jabs. Is this how coders are supposed to act?

**Comment by Rain7x** [comment]
I think they're supposed to write code

Looks like he did his job

**Comment by TheSilverNuke** [comment]
So grief code is good code got it. Got it got it I completely understand now.....

**Comment by TheVekter** [comment]
> What the hell has happened to TG

You know you're like... banned from the server discord for defending loli art, right?

**Comment by FancyCyclops** [comment]
That is not the point.

**Comment by actioninja** [comment]
image

**Pull request merged**
